modules/ROOT/pages/afm-monitoring-public-apis.adoc

```diff
@@ -66,9 +66,9 @@ To prevent tests run in public locations from consuming too many resources, AFM
 
 Tests are run at fixed intervals that you define in schedules. Each test is associated with a schedule. When you run tests from public locations, AFM sets limits on the following aspects of scheduling:
 
-* *Number of Schedules*: By default, your organization can run up to five schedules at one time for testing from public locations. If you reach the limit of five schedules and want to run more monitors simultaneously, contact your MuleSoft Customer Success Manager about raising your maximum.
+* *Number of Schedules*: By default, your organization can run up to five schedules simultaneously for testing from public locations. If you reach the limit of five schedules and want to run more monitors simultaneously, contact your MuleSoft Customer Success Manager about raising your maximum.
 
-* *Duration of Intervals Between Executions of Tests*: Account types of Employee can schedule intervals up to five minutes, and account type of Developer can schedule intervals up to 15 minutes. You can configure intervals using a cron expression.
+* *Duration of Intervals Between Executions of Tests*: You can schedule intervals of up to 15 minutes using a cron expression.
```

Comment:
I think this sentence `You can schedule intervals of up to 15 minutes using a cron expression` is not accurate because 15 minutes is the minimum interval allowed for some account type. However, you can configured bigger intervals like:
* One time per day, week, month, year
* One time on every Monday
* Every X hours
* Etc.

Reply:
Thanks, @dlarralde. I'll update this accordingly.